backend/controllers/EscapeRoomController.js

```diff
@@ -29,6 +29,15 @@ exports.storeEscapeRoom = async function (req, res) {
   res.status(200).json(result);
 }
 
+exports.getEscapeRoomById = async function (req, res) {
```

Comment:
Bij deze krijg ik de volgende error, `UnhandledPromiseRejectionWarning: SequelizeDatabaseError: Unknown column 'escapeRoomId' in 'field list'`
Waarschijnlijk moet je master nog even mergen in deze branche, daar heb ik het probleem volgens mij al opgelost.